Applications/Classification/jni/Android.mk

```diff
@@ -9,7 +9,9 @@ endif
 
 ifndef NNTRAINER_ROOT
 NNTRAINER_ROOT := $(LOCAL_PATH)/../../../libs/arm64-v8a
-NNTRAINER_INCLUDES := $(LOCAL_PATH)/../../../nntrainer/include
+NNTRAINER_INCLUDES := $(LOCAL_PATH)/../../../nntrainer/include \
+	$(LOCAL_PATH)/../../../api/capi/include \
```

Comment:
as far as I know, we are not using capi for android. why do we need capi includes here? do we need them?

Reply:
api/capi/incluide/platform is to provide ml-api-common.h which is used by nntrainer_error.h
api/capi/include is to provide nntrainer.h which is used by databuffer_func.h for data generator callback declaration